store/copr/coprocessor.go

```diff
@@ -70,13 +70,15 @@ const (
 	copNextMaxBackoff      = 20000
 	CopSmallTaskRow        = 32 // 32 is the initial batch size of TiKV
 	smallTaskSigma         = 0.5
+	smallConcPerCore       = 20
```

Comment:
Any experience or experiments about this choice?

Reply:
With the PR itself, this number seems a bit conservative to me. Maybe ok if this is combined with coprocessor batch.

Reply:
As the solution for the massive table lookup tasks, it's conservative, however, if we calculate the number of goroutines(given a 16c instance, there will be 320 goroutines per cop iterator, and there are 5 table workers in 1 index lookup executor), it's not a small number. The rest speedup operation might be done by coprocessor batch.